src/DocumenterCitations.jl

```diff
@@ -1,7 +1,28 @@
 module DocumenterCitations
 
+using Documenter
+using Documenter.Anchors
+using Documenter.Builder
+using Documenter.Documents
+using Documenter.Selectors
+using Documenter.Utilities
+using Documenter.Expanders
+
+using Markdown
+using Bibliography
+using Bibliography: xnames, xyear, xlink, xtitle, xin
+
+export CitationBibliography
+struct CitationBibliography <: Documenter.Plugin
+    bib # dictionary
+end
+function CitationBibliography(filename::AbstractString="")
```

Comment:
```suggestion
function CitationBibliography(filename="")
```

Reply:
We run into method ambiguity if we don't specify the type, so I had to add this for default behavior. Also, we need the outer constructor:
```
Any plugin that needs to either solicit user input or store information in a
[`Documents.Document`](@ref) should create a subtype of `Plugin`. The
subtype, `T <: Documenter.Plugin`, must have an empty constructor `T()` that
initialized `T` with the appropriate default values.
```

Reply:
Ah I see, didn't realize they were there for a reason. Please ignore my suggestions then!